man/systemd.network.xml

```diff
@@ -1121,6 +1121,41 @@ Table=1234</programlisting></para>
           Defaults to <literal>no</literal>.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname><replaceable>label</replaceable></term>
+        <listitem>
+
+          <para>This setting provides a method for integrating dynamic network configuration into Linux
+          <ulink url="https://docs.kernel.org/netlabel/index.html">NetLabel</ulink> subsystem rules, used by
+          <ulink url="https://en.wikipedia.org/wiki/Linux_Security_Modules">Linux security modules
+          (LSMs)</ulink> for network access control. The option expects a single NetLabel label. The label
+          must conform to lexical restrictions of LSM labels. When an interface is configured with IP
+          addresses, the addresses and subnetwork masks will be appended to the <ulink
+          url="https://github.com/SELinuxProject/selinux-notebook/blob/main/src/network_support.md">NetLabel
+          Fallback Peer Labeling</ulink> rules. They will be removed when the interface is
+          deconfigured. Failures to manage the labels will be ignored.</para>
+
+          <para>Warning: Once labeling is enabled for network traffic, a lot of LSM access control points in
+          Linux networking stack go from dormant to active. It is easy for someone not familiar with the LSM
```

Comment:
i'd not make this personal. i.e. remove "it is easy for someone not familiar with the blablablabla" and write instead something like "Care should be taken, not to blablablabla, because blablabla"

Reply:
Fixed